src/examples/forms/file-upload/Disabled.tsx

```diff
@@ -33,9 +33,9 @@ const Example = () => {
             {isDragActive ? (
               <span>Drop files here</span>
             ) : (
-              <span>Choose a file of drag and drop here</span>
+              <span>Choose a file or drag and drop here</span>
             )}
-            <Input {...getInputProps()} disabled />
+            <input {...getInputProps()} disabled />
```

Comment:
With this using a native `input` are the correct aria attributes being applied for the label to be read by assistive devices?

Reply:
I intended to call this out post-draft – I'm glad you caught it. Yes, this does break the association and needs to be reverted (and subsequently added to the `FileUpload` page imports).